contracts/truefi/TrueLender.sol

```diff
@@ -111,9 +112,32 @@ contract TrueLender is Ownable {
         pool.borrow(amount);
         currencyToken.approve(address(loanToken), amount);
         loanToken.fund();
+        loans.push(loanToken);
         emit Funded(address(loanToken), amount);
     }
 
+    function valueFor(ILoanToken loanToken) public view returns (uint256) {
+        (uint256 amount, uint256 apy, uint256 duration) = loanToken.getParameters();
+        uint256 start = loanToken.start();
+        uint256 passed = block.timestamp.sub(start);
+        if (passed > duration) {
+            passed = duration;
+        }
+
+        uint256 helper = amount.mul(apy).mul(passed).mul(loanToken.balanceOf(address(this)));
+        uint256 interest = helper.div(360 days).div(10000).div(loanToken.totalSupply());
+
+        return amount.add(interest);
+    }
+
+    function value() external view returns (uint256) {
```

Comment:
totalLoanValue?

Reply:
I'd stick to the simple, one word name on the interfaces.